README.rst

```diff
@@ -64,6 +64,7 @@ Option 2: Clone repo (recommended for developers)
     6) Install ``sup3r`` and its dependencies by running:
        ``pip install .`` (or ``pip install -e .`` if running a dev branch
        or working on the source code)
+    7) Set up the pre-commit hooks by calling ``pre-commit install``
```

Comment:
We should use `pip install pre-commit` to be consistent.

Reply:
Just to clarify, do we want to add pre-commit to the requirements.txt ?

Reply:
No I think we can leave this as an optional install.

Reply:
maybe note that ``pre-commit install`` is optional too